psbench/proxystore.py

```diff
@@ -30,35 +33,35 @@ def init_store_from_args(
     """
     store: Store | None = None
 
-    if args.ps_backend == STORES.ENDPOINT.name:
-        store = init_store(
-            STORES.ENDPOINT,
+    if args.ps_backend == 'ENDPOINT':
+        store = EndpointStore(
             name='endpoint-store',
             endpoints=args.ps_endpoints,
             **kwargs,
         )
-    elif args.ps_backend == STORES.FILE.name:
-        store = init_store(
-            STORES.FILE,
+        register_store(store)
```

Comment:
Can you move the `register_store(store)` to right before the return statement so there's just one?